includes/rest-api/Controllers/Version2/class-wc-rest-system-status-v2-controller.php

```diff
@@ -1172,22 +1172,27 @@ public function get_pages() {
 			_x( 'Shop base', 'Page setting', 'woocommerce' ) => array(
 				'option'    => 'woocommerce_shop_page_id',
 				'shortcode' => '',
+				'block'     => '',
 			),
 			_x( 'Cart', 'Page setting', 'woocommerce' ) => array(
 				'option'    => 'woocommerce_cart_page_id',
 				'shortcode' => '[' . apply_filters( 'woocommerce_cart_shortcode_tag', 'woocommerce_cart' ) . ']',
+				'block'     => apply_filters( 'woocommerce_cart_block_tag', 'woocommerce/cart' ),
```

Comment:
Thanks for the explanation @budzanowski, but I agree with @nerrad that we should hold off on this filter. Can we remove the filter before moving forward with this?

Reply:
Done.